integration/107_launch_with_ebpf_test.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+start_suite "Launch scope and check it boots, with ebpf connection tracking enabled"
```

Comment:
We will drop 107 (and use `endpoints_have_ebpf()` in the other two tests to check if ebpf is enabled).